websites/0-9/3388/presence.ts

```diff
@@ -0,0 +1,83 @@
+const presence = new Presence({
+	clientId: "1094931941616267344",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/h8N7txd.png",
+			smallImageKey: "https://i.imgur.com/5f3sdaH.png",
+			smallImageText: "Browsing...",
+		},
+		{ pathname } = document.location;
+
+	if (pathname.includes("/detail/")) {
+		const filmItems = document.querySelectorAll(".fd-item"),
+			coverElement = document.querySelector<HTMLElement>(".detail-film-cover");
+
+		let name = document.querySelector(".film-name").textContent;
+
+		if (
+			filmItems.length >= 3 &&
+			!Number.isNaN(parseInt(filmItems[2].textContent))
+		)
+			name += ` (${filmItems[2].textContent})`;
+
+		if (
+			coverElement.style.backgroundImage.startsWith('url("') &&
+			coverElement.style.backgroundImage.endsWith('")')
+		) {
+			presenceData.largeImageKey = coverElement.style.backgroundImage.slice(
+				5,
+				coverElement.style.backgroundImage.length - 2
+			);
+		}
+
+		presenceData.details = `Viewing ${
+			!document.querySelector("#season-list") ? "a Movie..." : "a TV Show..."
```

Comment:
Remove "..." from strings. (Also remove from the lines at the bottom of this)
```suggestion
			!document.querySelector("#season-list") ? "a Movie" : "a TV Show"
```